platform/osx/export/codesign.cpp

```diff
@@ -1208,7 +1208,7 @@ Error CodeSign::_codesign_file(bool p_use_hardened_runtime, bool p_force, const
 	String id;
 	String main_exe = p_exe_path;
 
-	DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	DirAccessRef da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
```

Comment:
Those were leaking too (FYI @bruvzg since this is recent code - `DirAccess *` and `FileAccess *` always need a `memdelete` before any return).